lib/ansiblelint/color.py

```diff
@@ -0,0 +1,18 @@
+"""Console coloring support."""
+from enum import Enum
+
+
+class C(Enum):
+    """Color styles."""
+
+    reset = "0"
+    error_code = "1;31"  # bright red
```

Comment:
This is problematic because it'll only work in terminals that support misusing the bold setting for bright. Standard-compliant terminals don't work like that.

Reply:
My change does not touch current behavior. ANSI support is a can of worms due to the variance in implementations. I am ok to investigate drop of bold but in a different change, also after looking at what other linters are doing with output (is likely that ansible-lint would be used in parallel with others too).